en/includes/guides/authorization/user-impersonation/user-impersonation.md

```diff
@@ -52,37 +52,115 @@ With the required permissions in place, the impersonator can now impersonate ano
 
     ![User Impersonate Button]({{base_path}}/assets/img/guides/authorization/impersonation/user-impersonate-button.png)
 
-4. You should now be logged into the **My Account** application as the impersonated user.
+4. Log into **My Account** application as the impersonated user.
 
     ![Discoverable Applications]({{base_path}}/assets/img/guides/authorization/impersonation/impersonated-myaccount-no-applications.png)
 
 5. The **My Account** portal shows all business applications that the impersonator can access as the impersonated user. Launch any application to log in as that user.
 
-    !!! note "Display applications in the My Account portal"
+    !!! note "Display applications in the **My Account** portal"
 
-        To make an application discoverable in the My Account portal for impersonators, you need to both enable impersonation for the application and mark it as discoverable. See [Configure business applications for impersonation](#configure-business-applications-for-impersonation) for instructions.
+        To make an application discoverable in the **My Account** portal for impersonators, you need to both enable impersonation for the application and mark it as discoverable. See [Configure business applications for impersonation](#configure-business-applications-for-impersonation) for instructions.
+
+## Impersonate users in a sub organization
+
+Sub organization administrators also impersonate users within their organization by following these steps:
```

Comment:
We do not use the term "sub organization" in documentation. Please change the term according to the following:

Root organization - For the root.
Organization - For any sub org.
Child organization (if speaking in relative terms)
Parent organization (if speaking in relative terms)